pkgs/development/compilers/solc/default.nix

```diff
@@ -10,69 +20,98 @@ assert z3Support -> z3 != null && lib.versionAtLeast z3.version "4.6.0";
 assert cvc4Support -> cvc4 != null && cln != null && gmp != null;
 
 let
-  jsoncppVersion = "1.9.4";
+  jsoncppVersion = "1.9.3";
```

Comment:
Did you forgot to clean up this while doing experimentation?

Reply:
no, this is the version that they use https://github.com/ethereum/solidity/blob/be5647735e55637ab8908dcf3b3324a6e9b1e46f/cmake/jsoncpp.cmake#L47

Reply:
Then should replace the url in line 26 by jsoncppUrl?

Reply:
it is, during the postPatch phase

Reply:
I mean, why do we want to patch jsoncpp to 1.9.4. Why don't we just download jsoncpp 1.9.3 in line 26.

`jsoncppVersion` is updated to `1.9.4` in #101873 , which does not look correct. (We won't be able to find 1.9.4 url in `postPatch` phase)

Reply:
I forgot to switch it. should be fixed now